TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithSelectDotStar.rdz.cbl

```diff
@@ -0,0 +1,11 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. DVZZMFT3.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+         EXEC SQL 
+        SELECT ensi.hend.laabidi.* , khaled.*
```

Comment:
Please add a two-part qualified name here so and check that we correctly capture all parts in the correct tokens.